docs/clients.md

```diff
@@ -0,0 +1,100 @@
+# Clients
+
+**Lumberjack is not an official Google product.**
+
+Lumberjack client consists of the following processors and they run in order:
+
+1.  Validator(s): To validate audit log requests
+2.  Mutator(s): To mutate audit log request, e.g. applying common labels
```

Comment:
Maybe adding a "the" here to make it "To mutate the audit log request," is a good idea.